### PR TITLE
fix #55314 blutv.com

### LINKED ISSUE #55314: blutv.com
### Issue URL (Annoyance)
 [https://www.blutv.com/film](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.blutv.com%2Ffilm)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | iOS
AdGuard version: | 2.1.2
Browser: | Safari
System wide filtering: | disabled
Simplified filters: | disabled
AdGuard DNS: | Default
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Other:</b><br/> AdGuard Safari<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#55314

### CODE REVIEW

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -91,6 +91,7 @@ sodapdf.com##.be-tooltip
 !
 nejm.org#$#.featherlight { display: none !important; }
 nejm.org#$#html { overflow: visible !important; }
+blutv.com##.ant-btn
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/popups.txt:94]: Is it still required? Cannot find.

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/popups.txt:94]: "Register now"
<details>
<summary>Screenshot</summary>

</details>

Comment (Alex-302) [reply AnnoyancesFilter/sections/popups.txt:94]: This rule breaks register form.

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/popups.txt:94]: Ok, I see. So it can not be blocked safely on iOS.

**Pull request closed without merging**